doc/nrf/ug_creating_matter_accessory.rst

```diff
@@ -0,0 +1,267 @@
+Creating Matter Sensor application from Matter Template sample
+##############################################################
+
+| In this tutorial you will learn how to create your own device application using Matter. 
+| We will show a way to modify Matter Template sample in order to obtain a sensor application.
```

Comment:
I will let @greg-fer comment on that, however I think that we shouldn't use in documentation personal form, like "we", "our" and etc.

Reply:
Yes, I think that would be more aligned with other docs.